gnd/build.gradle

```diff
@@ -205,7 +205,3 @@ apply plugin: 'com.google.gms.google-services'
 task checkCode(type: GradleBuild) {
     dependsOn 'checkstyle', 'lintDebug', 'pmd', 'spotbugs'
 }
-
-afterEvaluate {
```

Comment:
This seems unrelated to the overall PR--what's the impetus for this change?

Looks like this lets us assemble a debug build w/o enforcing style etc. checks?

LGTM, just calling it out in case it slipped into this PR accidentally.

Reply:
Actually it did! I removed it because it was slowly down development significantly, but we can discuss this separately. Added it back.